settings/src/components/password.js

```diff
@@ -40,6 +40,9 @@ export default function Password() {
 	const [ inputType, setInputType ] = useState( 'password' );
 	const [ hasAttemptedSave, setHasAttemptedSave ] = useState( false );
 	let passwordStrong = true; // Saved passwords have already passed the test.
+	// Originally, password wasn't set in record, but we need it to be an empty string for 
+	// resetting to initial after navigating away from password setting page.
+	userRecord.record.password = '';
```

Comment:
Should this be here or in `utilities.js`: https://github.com/WordPress/wporg-two-factor/blob/31cc878ad267049b8f0ecda17a80a97bd7bc7aaf/settings/src/utilities.js#L7

Reply:
I think both would work, but I'm tempted to put it in `password.js`, as
1. the `record.password` value would always be empty for the new password input here.
2. it's easier to locate it and do some tweaks once we decide to add an `old password` to the view - so that users have to verify the old password before changing to a new one.
3. as above said, if we're going to add an old password, it would be more appropriate to change it to userRecord.record.newPassword here.
4. there would never be a newPassword state in `userRecord.record`. (there might be record.password in the future if we decide to add in the old password)